src/Elcodi/Component/User/Entity/Abstracts/AbstractUser.php

```diff
@@ -38,6 +38,21 @@
         DateTimeTrait,
         EnabledTrait;
 
+    const GENDER_UNDEFINED = 0;
```

Comment:
Undefined? Can someone have undefined sex? Shouldn't be this null? And being null, this wouldn't be necessary, right?

By default, null (undefined), and each factory should define the default value